android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt

```diff
@@ -70,6 +90,17 @@ val darkerGrayButtonColor
 private val darkerGrayButtonColorLight = Color((0xFFE5E5E5))
 private val darkerGrayButtonColorDark = Color((0xFFE5E5E5))
 
+val lightGrayButtonColor
+	@Composable
+	get() = if (isSystemInDarkTheme()) {
+		lightGrayButtonColorDark
+	}
+	else {
+		lightGrayButtonColorLight
+	}
+private val lightGrayButtonColorLight = Color((0xFFEEEEEE))
+private val lightGrayButtonColorDark = Color((0xFFEEEEEE))
```

Comment:
With our new dark mode updates, this should now be working. Please make sure that the dark and light mode colors both work on your PR.